### DISCUSSION TITLE
can base domain default to public page?

### DISCUSSION THREAD

**Discussion by agp10** [AzuraCast/AzuraCast#5310] [Q&A] [answered]
Hello, is it possible to configure my base domain to route to my stations' public page?

-example-
typing: myradio.com
would load: myradio.com/public/radio

Currently, the base url loads the azuracast login page.

typing: myradio.com
loads: myradio.com/login

having the public page load by default instead of the admin login would be nice

Thank you

**Comment by BusterNeece** [comment, marked as answer]
@agp10 There's a setting in the Custom Branding settings that lets you specify a redirect for the root path (`/`) on the domain. Once you've set up the redirect, you can still directly visit `/login` to access the interface.

**Reply by nstrvntn** [reply]
If you know, can you tell me if I can do it not by redirect, but so that my domain opens my radio station?
